gen/service.go

```diff
@@ -23,10 +23,9 @@ package gen
 import (
 	"bytes"
 	"fmt"
+	"go.uber.org/thriftrw/compile"
```

Comment:
nit: imports formatting here should look like previous

Reply:
> Please add a more detailed title!

Apologies! I mixed up the location box of the branch name and the title name! The commit message title has been moved to the correct place in the github UI!